### ISSUE TITLE
bert_history = model.fit(ds_train_encoded, epochs=number_of_epochs, validation_data=ds_test_encoded)

### ISSUE THREAD

**Issue by francespizzuo** [srinath1882/ClassifyCompaniesByIndustry#1]
The command takes way too much time. ETA is 9:24:51 and keeps going up. While running:
''''
Epoch 1/8
   2/1985 [..............................] - ETA: 9:24:51 - loss: 4.1392 - accuracy: 0.0000e+00
''''

**Comment by srinath1882** [comment]
Hi.. Are you running it on a GPU? You can try running it on Google Colab, where it takes almost 5 mins for an epoch.

**Comment by francespizzuo** [comment]
Tried running on Google Colab. It works faster. Thanks!
Please, also note that: model = TFBertForSequenceClassification.from_pretrained("bert-base-uncased", config = config, training=training) returns that training is not defined. So you might as well consider deleting this args (at least for my python 3+ version).
